docker-compose.yml

```diff
@@ -390,3 +392,20 @@ services:
       JAVA_OPTS: -Dfile.encoding=UTF-8
     volumes:
       - georchestra_datadir:/etc/georchestra
+      
+  rabbitmq:
+    image: docker.io/bitnami/rabbitmq:3.12
+    ports:
+      - '4369:4369'
```

Comment:
do we really need to expose by default any rabbitmq ports? the applications that are going to use rabbitmq are already inside docker, exposing sensitive ports is not a good idea security wise.